tripy/tripy/backend/compiler_api.py

```diff
@@ -33,6 +33,110 @@
 from tripy.utils import json as json_utils
 
 
+@export.public_api(document_under="compiler")
+def synchronize():
+    """
+    Synchronizes the default stream.
+
+    .. code-block:: python
+        :linenos:
+        :caption: Use streams in execution
```

Comment:
Since there's only one:
```suggestion
        :caption: Example
```